### ISSUE TITLE
Front matter text clarifications for the "Collective Communication" and "Topology" Chapters

### ISSUE THREAD

**Issue by tonyskjellum** [mpi-forum/mpi-issues#90]
# Problem

Clarifications are needed now that Persistent Communication has been added.  

# Proposal

Persistent communication addition has raised the need for clarifying text.  This definitely applies to the Collective chapter, and there may be incremental text also needed in the Topology chapter.

# Changes to the Text

TBD.

# Impact on Implementations

None.

# Impact on Users

Clarity for users.

# References

Ticket #25 is driving this requirement, but we want to settle this concern separately as agreed in Forum meetings.

**Comment by tonyskjellum** [comment]
A review was done on June 13, 2018 at the MPI meeting in Austin.  Updates will be made an a reading will be presented in Barcelona.

**Comment by tonyskjellum** [comment]
@puribangalore --- Please look at Ticket #80; the latest update adds _X APIs for the persistent collectives; in the process, I've had to go back and enumerate both persistent and persistent _X operations where they have failed to be named throughout Chapter 5.  This has got to be related to editorial changes you've been undertaking or plan to do here.  In fact, we need to correlate those changes.  Ticket #105 will likewise do a review of function enumeration once we add the _X APIs later this week.

**Comment by tonyskjellum** [comment]
@puribangalore --- we need to update this for the meeting next Thursday ... we need to finish before November 10! We will present in San Jose... do you need help?

**Comment by tonyskjellum** [comment]
Here is the latest text, before the two-week deadline, so we can read the ticket.  As soon as we fix a make/latex issue with the latest mpi-3.x checkout, the pull request will match this.  [I can't build right now, @puribangalore  built this from India but could not upload.]

[mpi32-report-ticket90-19nov18.pdf](https://github.com/mpi-forum/mpi-issues/files/2596108/mpi32-report-ticket90-19nov18.pdf)

**Comment by tonyskjellum** [comment]
Here's the text we actually want you to read for the two-week deadline review:

[mpi32-report-ticket90-19nov18.pdf](https://github.com/mpi-forum/mpi-issues/files/2596168/mpi32-report-ticket90-19nov18.pdf)

**Comment by tonyskjellum** [comment]
@puribangalore read the chapter; there was various feedback of a general nature.

For instance, issues with commas and capitalization (already addressed to first order).

However, by strawpoll, we voted to reorganize the chapter so that "All Barrier, "All Broadcast," ... "All Exscan." is the order, rather than a parallel section Blocking, Nonblocking, and Persistent.  That will necessarily "transpose the 3 sections of operations, and move ahead the discussions, so that the operations will all be "together." This mirror the topologies and other chapters better.   This is meant to be part of this ticket... and a careful review is required to be sure we don't change any meanings.  That is not supposed to happen just by reordering presentation of the APIs.  @puribangalore and @tonyskjellum will do this reordering task and present in Chattanooga in March.

**Comment by tonyskjellum** [comment]
We did not do the recommended steps for Chattanooga.  We are revisiting this task for Chicago.

**Comment by tonyskjellum** [comment]
Aiming for MPI-4.1 because it is "text tidying."  We will wait till all MPI-4.0 work is done before revisiting this.

**Comment by Wee-Free-Scot** [comment]
This needs to move very quickly if it is going to hit MPI-4.1 -- who is working on it?

**Comment by wesbland** [comment]
This wasn't read at the December 2022 meeting. The last opportunity for MPI 4.1 is to have it ready at the March 2022 meeting (and it needs to "pass" the reading).

@tonyskjellum / @puribangalore Are you (or is someone else) planning to push this forward or should we move it out of the plan for MPI 4.1?

**Comment by jaegerj** [comment]
This won't be ready for 4.1 but can be targeted to MPI 5.0.

The title does not fit the content of the ticket anymore.
Some clarifications have already been made, and another ticket exist to focus on the indroductory text of the collective chapter (https://github.com/mpi-forum/mpi-issues/issues/256 -- which also should be rename with the current title of this ticket ironically)

The main part of the ticket which has not been handled yet is the reorganizing of the collective chapter.
Maybe we should rename this ticket accordingly.

**Comment by wesbland** [comment]
@jaegerj - You're welcome to rename the ticket to anything you like. :)

**Comment by jaegerj** [comment]
@tonyskjellum @puribangalore would it be ok with you to rename this ticket "Reorganizing the collective chapter" ?

**Comment by puribangalore** [comment]
Fine with me.
